src/bitcoin-tx.cpp

```diff
@@ -725,7 +725,7 @@ static void MutateTx(CMutableTransaction& tx, const std::string& command,
 static void OutputTxJSON(const CTransaction& tx)
 {
     UniValue entry(UniValue::VOBJ);
-    TxToUniv(tx, uint256(), entry);
+    TxToUniv(tx, uint256(), entry, false); // TODO: bitcoin-tx doesn't go through the addresses deprecation phase currently, defaults to new behavior. Do we want to add a deprecation flag here?
```

Comment:
e2b814e315f011b6afea26a1ac10aa9f31b5fddb: This needs release notes 

```suggestion
    TxToUniv(tx, uint256(), entry, /* include_addresses */ false);
```